cmd/keytransparency-sequencer/main.go

```diff
@@ -231,11 +231,17 @@ func runSequencer(ctx context.Context, conn *grpc.ClientConn,
 	if err := signer.AddAllDirectories(ctx); err != nil {
 		glog.Errorf("runSequencer(AddAllDirectories): %v", err)
 	}
+
 	go sequencer.PeriodicallyRun(ctx, time.Tick(*refresh), func(ctx context.Context) {
 		if err := signer.AddAllDirectories(ctx); err != nil {
 			glog.Errorf("PeriodicallyRun(AddAllDirectories): %v", err)
 		}
-		if err := signer.RunBatchForAllMasterships(ctx, int32(*batchSize)); err != nil {
+		if err := signer.DefineRevisionsForAllMasterships(ctx, int32(*batchSize)); err != nil {
```

Comment:
Should we separate this from `AddAllDirectories`, to save a bit of latency?

Reply:
I think `AddAllDirectories` should also run with a more relaxed cadence, because directories are rarely (almost never) added.

Reply:
#1348